eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -9,7 +9,6 @@ parameters:
   TargetDocRepoName: ''
   TargetDocRepoOwner: ''
   PRBranchName: 'master-rdme'
-  SourceBranchName: 'master'
   PRLabels: 'auto-merge'
```

Comment:
After all the work is in go through these all these parameters remove any of the ones we are no longer using.